src/fragment-components/a-radiobutton-group.tsx

```diff
@@ -0,0 +1,200 @@
+			placeholder='Legend name'
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					legend: event.currentTarget.value,
+				});
+			}}
+		/>
+		<Dropdown
+			label='Orientation'
+			titleText='Orientation'
+			items={orientationItems}
+			initialSelectedItem={orientationItems.find(item => item.id === selectedComponent.orientation)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				orientation: event.selectedItem.id
+		})}/>
+		<Dropdown
+			label='Label position'
+			titleText='Label position'
+			items={labelPositions}
+			initialSelectedItem={labelPositions.find(item => item.id === selectedComponent.labelPosition)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				labelPosition: event.selectedItem.id
+		})}/>
+		<ComponentCssClassSelector componentObj={selectedComponent} setComponent={setComponent} />
+	</>
+};
+
+export const ARadioButtonGroupCodeUI = ({ selectedComponent, setComponent }: any) => {
+	let allItems = selectedComponent.items.map((item: any) => {
+		return {
+			text: item.labelText,
+			id: item.id,
+			defaultChecked: item.defaultChecked
+		}
+	});
+	selectedComponent.defaultSelected = `${selectedComponent.items.find(((item: any) => {
+		return item.defaultChecked	
+	})).id}`;
+	
+	return <>
+		<Dropdown
+			label='Default selection'
+			titleText='Default selection'
+			items={allItems}
+			initialSelectedItem={allItems.find((item: any) => item.defaultChecked)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				valueSelected:`${event.selectedItem.id}`,
+				items: selectedComponent.items.map((item: any) => ({
+						...item, 
+						defaultChecked: event.selectedItem.id === item.id ? true : false
+				}))
+		})}/>
+	</>
+};
+
+
+export const ARadioButtonGroup = ({
+	children,
+	componentObj,
+	...rest
+}: any) => {
+return (
+		<AComponent
+		componentObj={componentObj}
+		{...rest}>
+			<RadioButtonGroup
+			className={componentObj.cssClasses?.map((cc: any) => cc.id).join(' ')}
+			legendText= {componentObj.legend}
+			disabled= {componentObj.disabled}
+			orientation={componentObj.orientation}
+			defaultSelected={componentObj.defaultSelected}
+			labelPosition={componentObj.labelPosition}
+			name={componentObj.codeContext?.name}>
+					{children}
+			</RadioButtonGroup>
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: ARadioButtonGroup,
+	styleUI: ARadioButtonGroupStyleUI,
+	codeUI: ARadioButtonGroupCodeUI,
+	keywords: ['radio', 'button', 'group'],
+	name: 'Radio buttons',
+	defaultComponentObj: {
+		disabled: false,
+		type: 'radioButtonGroup',
+		legend: 'Radio Button Group',
+		defaultSelected: '',
+		labelPosition: 'right',
+		orientation: 'horizontal',
+		items: [
+			{
+				type: 'radioButton',
```

Comment:
```suggestion
				type: 'radio',
```
?

Reply:
it throws this error when we change it to just radio. in nodeModules
<img width="1127" alt="Screen Shot 2022-04-12 at 2 38 40 PM" src="https://user-images.githubusercontent.com/23690711/163031018-b2116b78-af60-4729-aee0-32f68c432a57.png">
s